spec/spec_helper.rb

```diff
@@ -23,7 +23,7 @@
 
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
-  config.fixture_path = "spec/fixtures"
+  config.fixture_paths = ["spec/fixtures"]
```

Comment:
Similarly, I _believe_ this may only be supported in newer versions of Rails
```
> require "rspec/rails"
=> true
> RSpec.configure { |c| c.fixture_paths = ["spec/fixtures"] }
NoMethodError: undefined method `fixture_paths=' for #<RSpec::Core::Configuration:0x00007f9cfc9968d0
Did you mean?  fixture_path=
               fixture_path
               fixture_path?
from (pry):8:in `block in __pry__'
> RSpec.configure { |c| c.fixture_path = "spec/fixtures" }
=> "spec/fixtures"

```

Reply:
Yeah, good call! If this was being used for other things and outside of convertkit, I would take more care of these cases but going with the comment above, let's go with this for now.